### PR TITLE
Wip/organizers

### PR DESCRIPTION
**Pull request by SepidehAlassi**
add description about conference and organization

### CODE REVIEW

File: content/pages/2023/about.md [hunk 1]
```diff
@@ -6,32 +6,39 @@ section: euroscipy_2023
 template: page
 slug: about_2023
 ---
+### EuroSciPy Conference
+The EuroSciPy is an annual non-profit European conference on Scientific Python. It is a cross-disciplinary gathering focused on the use and
+development of the Python language in scientific research. This event strives
+to bring together both users and developers of scientific tools, as well as
+academic research and state-of-the-art industry.
```
Comment (glemaitre) [inline content/pages/2023/about.md:13]: Do you think that we should add the info Python VB in here or it should be in another page?

File: content/pages/2023/organizers.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+</br>
+</br>
+
+### Financial Aid and Diversity
+- Alessia Marcolini (LUMICKS, NL)
+- Milton Gomez (UNIL, CH)
+- Dr. Mojdeh Rastgoo (Saint-Gobain, FR)
+</br>
+</br>
+
```
Comment (glemaitre) [inline content/pages/2023/organizers.md:35]:
```suggestion

### Code of Conduct Committee
- Alessia Marcolini (LUMICKS, NL)
- Maren Westermann (Deutsche Bahn, DE)
- Dr. Mojdeh Rastgoo (Saint-Gobain, FR)
</br>
</br>

```

File: content/pages/2023/organizers.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+### Administration
+- Alexander CS Hendorf (Königsweg GmbH, DE)
+- Dr. Martin Christen (University of Applied Sciences and Arts Northwestern Switzerland, CH)
+</br>
+</br>
+
+### Supporting Institution
+[Python Software Verband e.V.](https://python-verband.org/)
+
```
Comment (glemaitre) [inline content/pages/2023/organizers.md:55]: Just to remove a bit of space

```suggestion
```

File: content/pages/2023/sprint.md [hunk 4]
```diff
@@ -36,3 +36,15 @@ List of confirmed sprints:
...
+#### Sprints Location
+
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2692.534226190979!2d7.58297061570638!3d47.55739329936902!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4791b9abfe2d63a7%3A0xb44b4a31a53b38fe!2sRosshofgasse%202%2C%204051%20Basel%2C%20Switzerland!5e0!3m2!1sen!2suk!4v1657383709427!5m2!1sen!2suk" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+&nbsp;
+
+####Address
+
+Roshoffgasse 2,
+
+CH-4051 Basel, Switzerland
```
Comment (glemaitre) [inline content/pages/2023/sprint.md:50]: Adding a bit more spacing.

```suggestion
<br>

### Sprints Location

#### Address

Roshoffgasse 2,<br>CH-4051 Basel,<br>Switzerland

#### Map

<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2692.534226190979!2d7.58297061570638!3d47.55739329936902!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4791b9abfe2d63a7%3A0xb44b4a31a53b38fe!2sRosshofgasse%202%2C%204051%20Basel%2C%20Switzerland!5e0!3m2!1sen!2suk!4v1657383709427!5m2!1sen!2suk" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
&nbsp;
```

**Review by glemaitre — APPROVED** [review]
Looks good on my side. I just did a couple of suggestions after checking the rendering when building the website. The only non-trivial changes proposed:

- add the CoC committee
- question-related to mentioning Python VB somewhere (I don't know what is best for that)

File: content/pages/2023/about.md (hunk 1 above)
Comment (SepidehAlassi) [reply content/pages/2023/about.md:13]: I will add a sentence about Python SV here too.

**Comment by SepidehAlassi** [comment]
> Looks good on my side. I just did a couple of suggestions after checking the rendering when building the website. The only non-trivial changes proposed:
> 
> * add the CoC committee
> * question-related to mentioning Python VB somewhere (I don't know what is best for that)

Thanks for reviewing this!

**Pull request merged**